Advanced_Projects/File_Organizer_GUI/filegui.py

```
@@ -0,0 +1,322 @@
+                    if self.copy_files.get():
+                        shutil.copy2(file_path, dest_path)
+                        action = "Copied"
+                    else:
+                        shutil.move(file_path, dest_path)
+                        action = "Moved"
+                    
+                    self._queue.put(lambda f=filename, d=dest_folder, a=action: self.log_message(f"{a}: {f} → {d}"))
+                    organized_count += 1
+                    
+                except Exception as e:
+                    self._queue.put(lambda f=os.path.basename(file_path), err=str(e): self.log_message(f"Error processing {f}: {err}"))
```

Ruff (0.11.9)

Do not perform function call `os.path.basename` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)